conan/api/subapi/new.py

```diff
@@ -14,6 +16,69 @@ class NewAPI:
     def __init__(self, conan_api):
         self.conan_api = conan_api
 
+    def save_template(self, template, defines=None, output_folder=None, force=False):
```

Comment:
All this is just moved from the Command CLI layer to be done in the API, no changes in logic.